models/Team.js

```diff
@@ -1,10 +1,12 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+const playerSchema = require('./Player').schema;
+const userSchema = require('./User').schema;
 
 const teamSchema = new Schema({
   name: String,
-	owner: String,
-	players: [String]
+	owner: [userSchema],
+	players: [playerSchema]
```

Comment:
Just to clarify: assuming the owner is playing on the team, can we assume that they will also be listed under the player field as well as the owner field?